pkg/sdk/renderer/argo/testdata/hub/implementation/bitnami/postgresql/install-v2.yaml

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
I took the newest manifests and add a `v2` prefix. I created [issue](https://github.com/capactio/capact/issues/712) to mitigate/track this problem.